packages/babel-plugin/src/utils/resolve-binding.ts

```diff
@@ -10,6 +10,7 @@ import resolve from 'resolve';
 import { DEFAULT_CODE_EXTENSIONS } from '../constants';
 import type { Metadata } from '../types';
 
+import { evaluateExpression } from './evaluate-expression';
```

Comment:
I think this is a cyclic import as `evaluate-expression` imports `resolveBinding` too.

Reply:
I too am surprised this works...

Reply:
good catch! I'll look for a different solution

Reply:
One way I can think of is to pass in `evaluate-expression` as a param to `resolveBinding`. But this may cause cyclic imports in other places where `resolveBinding` is used.

Reply:
This should be fixed now, I should have thought to use `traverseMemberExpression` in the first place!